apis/components/v1alpha1/datasciencepipelines_types.go

```diff
@@ -53,7 +53,8 @@ type DataSciencePipelinesSpec struct {
 }
 
 type DataSciencePipelinesCommonSpec struct {
-	common.DevFlagsSpec `json:",inline"`
+	common.DevFlagsSpec    `json:",inline"`
+	ModelRegistryNamespace string `json:"modelRegistryNamespace,omitempty"`
```

Comment:
Sorry that I wasn't clear in the Jira description, but we don't need a new field on the DSP CR but rather we just need the data passed to params.env like it was done for `PLATFORMVERSION`.

Reply:
correct. 
the way how release version gets done is actually getting value from DSCI not DSC. 
Since DSP does not watch DSC,  in order to update with new value if user change namespace, i will need signal DSP CR in order to write value in params.env.

Reply:
Does a change in value not result in an environment variable change on the DSPO deployment? If yes, then DSPO will restart which will cause a reconcile.

Reply:
That's depend on how the value is taken into account by the DSPO, the operator just put it in the `params.env` then it is the kustomization logic of the DSPO that maps it

Reply:
so from ModelReg side, user should not randomly change that namespace.
the only way to do that, is to delete DSCI and re-create new one with new ModelReg namespace.
in reality, a different value set in DSPO params.env is not that likely.

but if that really happens one day, as Luca said, it is totally up to how DSPO handles deployment by kustomize if deployments get updated or not.